specifications/api/swagger.yaml

```diff
@@ -125,6 +125,11 @@ x-blockReference: &blockReference
     time: *date
     height: *numberOfBlocks
 
+x-networkTag: &networkTag
+  description: Can be null for 'Icarus' and 'Byron' styles.
+  type: integer
+  minimum: 0
```

Comment:
This should not be necessary because the server is already instantiated for one of the given networks. So the network tag is implicit based on how one started the server (`--mainnet` or `--testnet`)

Reply:
removed in https://github.com/input-output-hk/cardano-wallet/pull/2253/commits/ccd4eb486e7739d6a71497891baf70b0f78eff86